backend/config.go

```diff
@@ -122,3 +123,26 @@ func (c *GrafanaCfg) AppURL() (string, error) {
 	}
 	return url, nil
 }
+
+type userAgentKey struct{}
+
+// UserAgentFromContext returns user agent from context.
+func UserAgentFromContext(ctx context.Context) (*useragent.UserAgent, error) {
+	v := ctx.Value(userAgentKey{})
+	if v == nil {
+		return useragent.New("0.0.0", "unknown", "unknown")
```

Comment:
I think this behaviour makes the most sense considering we wouldn't really want to return an error here - as older client versions (<`10.2.0`) won't send a user agent 👍 

Nit: Maybe it makes sense to have a `useragent.Empty()` here which will use this info. WDYT?

Reply:
@wbrowne That's an excellent idea! Added in cf7cfa8 🚀